tests/ray/test_dask.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+from typing import Any
+
+import dask.dataframe as dd
```

Comment:
you prob need to also add dask to our dev dependancies

Reply:
Ah good catch!